### ISSUE TITLE
Some files still reference the old PATENTS file

### ISSUE THREAD

**Issue by pombredanne** [facebook/rocksdb#5324]
Text such as :
```
 An additional grant
//  of patent rights can be found in the PATENTS file in the same directory.
```
is still present in some notices, even though this PATENTS file is now gone.

See: https://github.com/facebook/rocksdb/search?q=patents&unscoped_q=patents

- https://github.com/facebook/rocksdb/blob/cd43446d017fd3929e5883bccf1206afafd57952/util/crc32c_ppc.h
- https://github.com/facebook/rocksdb/blob/6c2bf9e916db3dc7a43c70f3c0a482b8f7d54bdf/util/ppc-opcode.h
- https://github.com/facebook/rocksdb/blob/6c2bf9e916db3dc7a43c70f3c0a482b8f7d54bdf/util/crc32c_ppc_constants.h
- https://github.com/facebook/rocksdb/blob/6c2bf9e916db3dc7a43c70f3c0a482b8f7d54bdf/util/crc32c_ppc_asm.S
- https://github.com/facebook/rocksdb/blob/6c2bf9e916db3dc7a43c70f3c0a482b8f7d54bdf/util/crc32c_ppc.c
- https://github.com/facebook/rocksdb/blob/a5e851e113ba66fd82b651fd0d3b13750fb75a55/port/win/win_jemalloc.cc

Found by @royaljust

**Comment by sagar0** [comment]
Thanks for brining this to our attention. I believe this is unintentional.
#5326 -- to fix this.

**Comment by sagar0** [comment]
Fixed in #5326.
